invenio_orcid/config.py

```diff
@@ -26,6 +26,12 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_AUTHORS_ELASTICSEARCH_INDEX_NAME = 'records-authors'
+ORCID_AUTHORS_ELASTICSEARCH_DOC_TYPE = 'authors'
+ORCID_MAIN_RECORDS_PID_TYPE_NAME = 'literature'
+ORCID_MAIN_RECORDS_DOC_TYPE = 'hep'
+ORCID_MAIN_RECORDS_IDENTIFIER = 'control_number'
```

Comment:
I am afraid I don't fully understand.

You mean this one:

https://github.com/inveniosoftware/invenio-pidstore/blob/master/invenio_pidstore/fetchers.py#L54

but this takes the `uuid` as an argument and in our case that's what we are trying to get using the `recid` which in inspire's case is called `control_number`

https://github.com/Panos512/invenio-orcid/blob/7904c68c2fdadadcc247e288160e76065a872c99/invenio_orcid/tasks.py#L51


If I understood correctly we need a fetcher that extracts the `recid` from a given record.